### PR TITLE
Adds extra information to objects

### PR DESCRIPTION
**Pull request by JDJGInc**
like worker, process id, and increment.
https://discord.com/developers/docs/reference#snowflakes-snowflake-id-format-structure-left-to-right
you get the extra information in docs here, for those who want lots of snowflake info.
if you don't like this, feel free to not add 🤷

## Summary

Adds extra stuff.
## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

I have utils for making a snowflake from a datetime object, or start from the current time using discord.py's datetime, but uh, I am not sure how many people would like it.

### CODE REVIEW

**Comment by Sengolda** [comment]
interesting changes, but in which cases do you think these properties will be used?

**Comment by Gnome-py** [comment]
Please rework/reword the docstrings in order to follow the rest of the doc format

File: discord/object.py [hunk 1]
```diff
@@ -94,3 +94,18 @@ def __repr__(self) -> str:
     def created_at(self) -> datetime.datetime:
         """:class:`datetime.datetime`: Returns the snowflake's creation time in UTC."""
         return utils.snowflake_time(self.id)
+    
+    @property
+    def worker_id(self) -> int:
+        """returns the worker id that made the user id when the user id was generate as :class:`int`: """
```
Comment (Sengolda) [inline discord/object.py:100]: These doc-strings should be something like this

```rst
Returns
--------
 :class:`int`
    the worker id that made the user id id when the user id was generated as an int.
```

Comment (Gnome-py) [reply discord/object.py:100]: No? Read literally one function up, see how it is 
""":class:\`datetime.datetime`: Returns the snowflake's creation time in UTC."""

**Comment by JDJGInc** [comment]
> interesting changes, but in which cases do you think these properties will be used?



> Please rework/reword the docstrings in order to follow the rest of the doc format

I have fixed the docs, and this meant for those who want extra snowflake info.

**Comment by JDJGInc** [comment]
I am sorry about this, for whatever reason I can not grab this pull request to change it, so I am going to try to reimplement the work I did in a different pull request.

**Pull request closed without merging**